routes/api/issues.js

```diff
@@ -249,16 +249,23 @@ router.get('/:issue_container/config.json.zip', async function (req, res) {
             console.log("Could not compress the file.")
                 return;
         }
+        //check if the ALLOW_CONFIG_DOWNLOAD is set to true
+        if(process.env.ALLOW_CONFIG_DOWNLOAD == 'false'){
```

Comment:
I would suggest to change this code so that it only executes when the `ALLOW_CONFIG_DOWNLOAD` is `true`. With how it's implemented now, if the developer forgets to set the env. variable, this code would actually expose the download functionality. That's not good.